client/src/App.js

```diff
@@ -77,6 +78,11 @@ function App() {
               component={Dashboard}
               isPrivateRoute
             />
+            <RouteContainer
+              path="/room/:path"
+              component={() => <WaitingRoom open={true} />}
+              isPrivateRoute
```

Comment:
This works for now but I'm wondering this will show on top of the dashboard or will just have the dialog?

Reply:
it just have the dialog